bad_code.java

```diff
@@ -8,14 +8,15 @@ public static void main(String[] args) {
```

Comment:
Consider using a code formatter to ensure consistent formatting and indentation. [medium]